### ISSUE TITLE
Option to use npm ci instead of npm install

### ISSUE THREAD

**Issue by bluec** [capistrano/npm#24]
`npm ci` would speed things up during deployments

https://docs.npmjs.com/cli/ci

**Comment by bluec** [comment]
Should also be noted that using `install` is not safe for deployments as it doesn't utilise the `package-lock.json`

Is anyone around to review and/or accept the PR? Maybe @swalkinshaw ?

**Comment by swalkinshaw** [comment]
I made a small comment but if the author isn't around I'll just merge anyway soon.

**Comment by swalkinshaw** [comment]
Feel free to do a separate PR after #23 to propose changing the default.

**Comment by bluec** [comment]
Thanks @swalkinshaw I have added a new PR #25

**Comment by swalkinshaw** [comment]
Closed by https://github.com/capistrano/npm/pull/23
